package.json

```diff
@@ -10,8 +10,9 @@
     "email": "info@ddem.us"
   },
   "main": "index.js",
-  "engine": {
-    "node": "12.14.1"
+  "engines": {
+    "node": "^14.17.1",
```

Comment:
I've been pinning this to the same version of Node that the Electron version we are using runs, so that if we run tests for the backend code, then they are running in the same environment as they do in the app.

Reply:
Hmm I see. Let me try downgrading back to 12 and seeing if the new version of builder still works. I only upgraded because of the upgraded dependency (there was a warning on installation for it but may not be a strict one)

Reply:
Alright confirmed that upgrading node to 14 is necessary for the latest version of electron-builder (see https://github.com/electron-userland/electron-builder/issues/5953)

Reply:
ok, I'm fine with doing this engines upgrade. We don't currently have unit tests for the main process yet (the one that runs in Node 12.14.1 in electron), but if we do add them then we should ensure that we run the unit tests on Node 12.14.1, and keep the build process in Node 14.

These node versions are important because we have seen our stack break with patch versions of node, and we have not tested the Mapeo Core stack with Node 14 yet. We found that if we did not test with the exact version we're using at runtime, we got unexpected errors not caught by the tests.

At some point we need to update electron, which includes the Node@14 upgrade for the runtime, but we need to test Mapeo Core in Node 14 for that.

Reply:
Yeah that make sense. Definitely not ideal to just upgrade the version for one dep but may be necessary because the most recent compatible version that potentially addresses our original issue is [`22.11.3`](https://github.com/electron-userland/electron-builder/releases/tag/v22.11.3) but using the `{arch}` macro that it enables is broken in that version but fixed in [`22.11.4`](https://github.com/electron-userland/electron-builder/releases/tag/v22.11.4), which we can't use unless we upgrade to node 14 🙃